docs/topics/funding.md

```diff
@@ -115,36 +111,50 @@ REST framework continues to be open-source and permissively licensed, but we fir
-* Support for customizable exception handling.
-* Support for Django's PostgreSQL HStoreField, ArrayField and JSONField.
-* Templated HTML form support, including HTML forms with nested list and objects.
-* Internationalization support for API responses, currently with 27 languages.
-* The metadata APIs for handling `OPTIONS` requests and schema endpoints.
-* Numerous minor improvements and better quality throughout the codebase.
+## What future funding will enable
 
-This incredible level of return on investment is *only possible through collaboratively funded models*, which is why we believe that supporting our paid plans is in everyone's best interest.
+* Realtime API support, using WebSockets. This will consist of documentation and support for using REST framework together with Django Channels, plus integrating WebSocket support into the client libraries.
+* Better authentication defaults, possibly bringing JWT in to the core package.
+* Securing the community & operations manager position long-term.
+* Opening up and securing a part-time position to focus on ticket triage and resolution.
+* Paying for development time on building API client libraries in a range of programming languages. These would be integrated directly into the upcoming API documentation.
 
 Sign up for a paid plan today, and help ensure that REST framework becomes a sustainable, full-time funded project.
 
 ---
 
-## Freelancer plan
+## What our sponsors and users say
 
-This subscription is recommended for freelancers and other individuals with an interest in seeing REST framework continue to&nbsp;improve.
+> As a developer, Django REST framework feels like an obvious and natural extension to all the great things that make up Django and it's community. Getting started is easy while providing simple abstractions which makes it flexible and customizable. Contributing and supporting Django REST framework helps ensure its future and one way or another it also helps Django, and the Python ecosystem.
+>
+> &mdash; José Padilla, Django REST Framework Contributor
 
-If you are using REST framework as an full-time employee, consider recommending that your company takes out a [corporate&nbsp;plan](#corporate-plans).
+>  The number one feature of the Python programming language is its community. Such a community is only possible because of the Open Source nature of the language and all the culture that comes from it. Building great Open Source projects require great minds. Given that, we at Vinta are not only proud to sponsor the team behind DRF but we also recognize the ROI that comes from it.
+>
+> &mdash; Filipe Ximenes, Vinta Software
+
+>  It's really awesome that this project continues to endure. The code base is top notch and the maintainers are committed to the highest level of quality.
+DRF is one of the core reasons why Django is top choice among web frameworks today. In my opinion, it sets the standard for rest frameworks for the development community at large.
+>
```

Comment:
Last testimonial is from Hacker News. I have a username but not a "real name" so I left it out. Could either include the username or leave out that statement?